transform/snowflake-dbt/models/hightouch/product_campaigns/onprem_trial_request_facts.sql

```diff
@@ -20,10 +20,10 @@ with latest_server_daily_details as (
         tr.email
         , tr.site_url
         , tr.license_id
-        , license_server_fact.server_id
-        , license_server_fact.start_date
-        , license_server_fact.expire_date
-        , license_server_fact.edition
+        , tr.server_id
+        , tr.start_date::date as start_date
+        , tr.end_date::date as expire_date
+        , 'E20 Trial' as edition
```

Comment:
3/5 this column is not needed downstream. Let's remove it.

Reply:
dropped this column